Source: patch from gentoo (flameeyes@gentoo.org)
Upstream: no, upstream is dead
Reason: patch to fix building with --as-needed, Gentoo bug #225211
Index: fcitx-scm/src/Makefile.am
===================================================================
--- fcitx-scm.orig/src/Makefile.am
+++ fcitx-scm/src/Makefile.am
@@ -32,5 +32,4 @@ fcitx_SOURCES = IC.c IC.h \
		fcitx.h extra.c extra.h \
		internalVersion.c
		
-fcitx_LDADD = ../lib/libXimd.a
-fcitx_LDFLAGS = -lXpm
+fcitx_LDADD = ../lib/libXimd.a -lXpm
